cves/CVE-2016-4461.yml

```diff
@@ -149,12 +164,24 @@ interesting_commits:
     emerging themes?
     If there are no interesting commits, demonstrate that you completed this section
     by explaining what happened between the VCCs and the fix.
-  answer:
+  answer: the following are interesting due to ther role in preventing this vulnerability from reoccuring.
   commits:
-    - commit:
-      note:
-    - commit:
-      note:
+    - commit: 02b207fca529627f3a34660de86b51444461a7d2
+      note: Allows redine of regex used to match methods when SMI is disabled.
```

Comment:
I think "redine of regex" is meant to be "redefine regex".

Reply:
Nice catch.